objects/ArkhamSCE380-5122024-Page1.bd6b3e.json

```diff
@@ -40,10 +40,10 @@
         "g": 1,
         "r": 1
       },
-      "Description": "- Added a new card to the Fan-made Accessories barrel that can be used to easily track bonus VP that can't immediately be spent.\n- The \"Take Clue\" hotkey can now be used to take a clue from a clue pool bag, and has variants for multi-handed play.\n- Opening a menu (such as the downloads menu) no longer causes it to appear on every other player's screen.\n",
+      "Description": "Draw Encounter button now draws to the rightmost empty threat area slot on left-click, and to draw area on right-click.\nCard states can now be changed with number keys.\r\nCleanup Helper now resets activation tokens.\r\nThe deck importer now can import Hemlock Vale story assets.\r\nHotkey Reference card has been updated.",
```

Comment:
The description for the encounter card drawing is still flipped 😅
You're describing the original plan, but then we had the group discussion to flip it so that the "left-click" functionality is untouched.